docs/notebooks/plot_slopes.ipynb

```diff
@@ -0,0 +1,2503 @@
+{
```

Comment:
<p>The first time I saw this I understood that in the previous case no eps was being added. Then I understood that it always add the eps. The difference is that now the eps is added to every value, while before it was added for the single value that was passed. Maybe this clarification can be changed. I'll read it again tomorrow.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/701/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>For continuous variables, <code>eps</code> is always added to each value. If the value of the dict is a single value, <code>eps</code> is added. If the value of the dict is a list or np.ndarray, <code>eps</code> is added to each element.</p>